### PR TITLE
Santa barbara county california

### PR DESCRIPTION
**Pull request by chunder**
Added Santa Barbara County, California.

### CODE REVIEW

File: scrapers/USA/CA/santa-barbara-county.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    cases += parse.number(
+      $('div.elementor-counter-title:contains("SANTA BARBARA COUNTY")')
+        .prev()
+        .find('span.elementor-counter-number')
+        .text()
+    );
+
+    return {
+      cases,
+      tested
```
Comment (lazd) [inline scrapers/USA/CA/santa-barbara-county.js]: Omit tested completely if you don't have data -- otherwise we're reporting that SB tested 0 people

**Comment by qgolsteyn** [comment]
Wondering if changes to Ventura County are intentional, maybe old code from #169 ?

**Comment by chunder** [comment]
I just pushed a revert commit to undo the changes to ventura-county.js. I must have fat fingered them into this branch somehow. This branch is only intending to create a new scraper for Santa Barbara County.

**Comment by qgolsteyn** [comment]
Closed as #176 was merged instead

**Pull request closed without merging**